### ISSUE TITLE
Matplotlib installation fails with Python 3.8

### ISSUE THREAD

**Issue by StrikerRUS** [ContinuumIO/anaconda-issues#11514]
### Actual Behavior

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: - 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
Examining python:   0%|                                  | 0/64 [00:00<?, ?it/s]
Comparing specs that have this dependency:   0%|          | 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0%|                             | 0/7 [00:00<?, ?it/s]
Finding shortest conflict path for python-dateutil:   0%| | 0/7 [00:00<?, ?it/s]
Finding shortest conflict path for python[version='>=3.7,<3.8.0a0']:  14%|▏| 1/7 [00:00<00:00, 28.92it/s]
Finding shortest conflict path for python=3.8:  29%|▎| 2/7 [00:00<00:00, 29.73it/s]                      
Finding shortest conflict path for python[version='>=2.7,<2.8.0a0']:  43%|▍| 3/7 [01:28<01:57, 29.48s/it]
Finding shortest conflict path for python[version='>=2.7,<2.8.0a0']:  57%|▌| 4/7 [01:28<01:06, 22.11s/it]
Finding shortest conflict path for python[version='>=3.5,<3.6.0a0']:  57%|▌| 4/7 [01:28<01:06, 22.11s/it]
Finding shortest conflict path for python[version='>=3.6,<3.7.0a0']:  71%|▋| 5/7 [01:28<00:44, 22.11s/it]
Finding shortest conflict path for python:  86%|█▋| 6/7 [01:28<00:22, 22.11s/it]                         
Comparing specs that have this dependency:  50%|█ | 1/2 [01:33<01:33, 93.82s/it]
Finding conflict paths:   0%|                             | 0/2 [00:00<?, ?it/s]
Finding shortest conflict path for python[version='>=3.8,<3.9.0a0']:   0%| | 0/2 [00:00<?, ?it/s]
Finding shortest conflict path for python=3.8:  50%|▌| 1/2 [00:00<00:00, 4928.68it/s]            
Examining certifi:  38%|█████████               | 24/64 [01:33<31:16, 46.91s/it]
Comparing specs that have this dependency:   0%|          | 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0%|                             | 0/4 [00:00<?, ?it/s]
Finding shortest conflict path for ca-certificates:   0%| | 0/4 [00:00<?, ?it/s]
Finding shortest conflict path for certifi[version='>=2016.09']:  25%|▎| 1/4 [00:05<00:16,  5.43s/it]
Finding shortest conflict path for certifi[version='>=2016.09']:  50%|▌| 2/4 [00:05<00:05,  2.71s/it]
Finding shortest conflict path for certifi:  50%|▌| 2/4 [00:11<00:05,  2.71s/it]                     
Finding shortest conflict path for certifi:  75%|▊| 3/4 [00:11<00:03,  3.80s/it]
Finding shortest conflict path for certifi[version='>=2016.9.26']:  75%|▊| 3/4 [00:17<00:03,  3.80s/it]
Finding shortest conflict path for certifi[version='>=2016.9.26']: 100%|█| 4/4 [00:17<00:00,  4.51s/it]
Comparing specs that have this dependency:  50%|█ | 1/2 [00:23<00:23, 23.89s/it]
Finding conflict paths:   0%|                             | 0/3 [00:00<?, ?it/s]
Finding shortest conflict path for ca-certificates:   0%| | 0/3 [00:00<?, ?it/s]
Finding shortest conflict path for certifi[version='>=2016.09']:  33%|▎| 1/3 [00:00<00:00, 280.97it/s]
Finding shortest conflict path for certifi[version='>=2016.9.26']:  67%|▋| 2/3 [00:00<00:00, 202.96it/s]
Examining wheel:  41%|██████████▌               | 26/64 [01:57<20:59, 33.14s/it]
Comparing specs that have this dependency:   0%|          | 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0%|                             | 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for wheel:   0%|           | 0/1 [00:00<?, ?it/s]
Comparing specs that have this dependency:  50%|▌| 1/2 [01:50<01:50, 110.80s/it]
Finding conflict paths:   0%|                             | 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for wheel:   0%|           | 0/1 [00:00<?, ?it/s]
Examining setuptools:  45%|█████████▌           | 29/64 [03:48<23:13, 39.82s/it]
Comparing specs that have this dependency:   0%|          | 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0%|                             | 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for setuptools:   0%|      | 0/1 [00:00<?, ?it/s]
                                                                                
Finding conflict paths:   0%|                             | 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for setuptools:   0%|      | 0/1 [00:00<?, ?it/s]
Examining pip:  53%|██████████████▉             | 34/64 [03:48<19:54, 39.82s/it]
Comparing specs that have this dependency:   0%|          | 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0%|                             | 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for pip:   0%|             | 0/1 [00:00<?, ?it/s]
Comparing specs that have this dependency:  50%|█ | 1/2 [00:04<00:04,  4.93s/it]
Finding conflict paths:   0%|                             | 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for pip:   0%|             | 0/1 [00:00<?, ?it/s]
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Package wheel conflicts for:
matplotlib -> python[version='>=3.5,<3.6.0a0'] -> pip -> wheel
python=3.8 -> pip -> wheel
Package python conflicts for:
python=3.8
Package setuptools conflicts for:
python=3.8 -> pip -> setuptools
matplotlib -> setuptools
Package ca-certificates conflicts for:
matplotlib -> python[version='>=3.5,<3.6.0a0'] -> ca-certificates
python=3.8 -> openssl[version='>=1.1.1d,<1.1.2a'] -> ca-certificates
Package pip conflicts for:
matplotlib -> python[version='>=3.5,<3.6.0a0'] -> pip
python=3.8 -> pip
Package python-dateutil conflicts for:
matplotlib -> python-dateutil
Package certifi conflicts for:
python=3.8 -> pip -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
```

### Expected Behavior

Successful installation.

### Steps to Reproduce

```
conda create -q -y -n test_env python=3.8
source activate test_env

conda info

conda list --show-channel-urls

conda install -y -n test_env matplotlib
```

##### `conda info`

<details>

```
     active environment : test_env
    active env location : /home/travis/miniconda/envs/test_env
            shell level : 1
       user config file : /home/travis/.condarc
 populated config files : /home/travis/.condarc
          conda version : 4.8.0
    conda-build version : not installed
         python version : 3.7.4.final.0
       virtual packages : __glibc=2.27
       base environment : /home/travis/miniconda  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/travis/miniconda/pkgs
                          /home/travis/.conda/pkgs
       envs directories : /home/travis/miniconda/envs
                          /home/travis/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.0 requests/2.22.0 CPython/3.7.4 Linux/4.15.0-1040-gcp ubuntu/18.04.3 glibc/2.27
414
                UID:GID : 2000:2000
             netrc file : None
           offline mode : False

```

</details>

##### `conda list --show-channel-urls`

<details>

```

# packages in environment at /home/travis/miniconda/envs/test_env:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
ca-certificates           2019.11.27                    0    defaults
certifi                   2019.11.28               py38_0    defaults
libedit                   3.1.20181209         hc058e9b_0    defaults
libffi                    3.2.1                hd88cf55_4    defaults
libgcc-ng                 9.1.0                hdf63c60_0    defaults
libstdcxx-ng              9.1.0                hdf63c60_0    defaults
ncurses                   6.1                  he6710b0_1    defaults
openssl                   1.1.1d               h7b6447c_3    defaults
pip                       19.3.1                   py38_0    defaults
python                    3.8.0                h0371630_2    defaults
readline                  7.0                  h7b6447c_5    defaults
setuptools                42.0.2                   py38_0    defaults
sqlite                    3.30.1               h7b6447c_0    defaults
tk                        8.6.8                hbc83047_0    defaults
wheel                     0.33.6                   py38_0    defaults
xz                        5.2.4                h14c3975_4    defaults
zlib                      1.2.11               h7b6447c_3    defaults

```

</details>

**Comment by StrikerRUS** [comment]
The issue is still here 🙁 .

**Comment by shivampotdar** [comment]
Even I am facing the same issue
![image](https://user-images.githubusercontent.com/32351257/72797750-b7225500-3c67-11ea-8584-770f3f8dab3b.png)

**Comment by StrikerRUS** [comment]
Just checked it right now - seems to be fixed.

**Comment by StrikerRUS** [comment]
Oops! Still the issue on macOS.
```
     active environment : test_env
    active env location : /Users/travis/miniconda/envs/test_env
            shell level : 1
       user config file : /Users/travis/.condarc
 populated config files : /Users/travis/.condarc
          conda version : 4.8.1
    conda-build version : not installed
         python version : 3.7.4.final.0
       virtual packages : __osx=10.14.4
       base environment : /Users/travis/miniconda  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/travis/miniconda/pkgs
                          /Users/travis/.conda/pkgs
       envs directories : /Users/travis/miniconda/envs
                          /Users/travis/.conda/envs
               platform : osx-64
             user-agent : conda/4.8.1 requests/2.22.0 CPython/3.7.4 Darwin/18.5.0 OSX/10.14.4
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```
```
# packages in environment at /Users/travis/miniconda/envs/test_env:
#
# Name                    Version                   Build  Channel
ca-certificates           2019.11.27                    0    defaults
certifi                   2019.11.28               py38_0    defaults
libcxx                    4.0.1                hcfea43d_1    defaults
libcxxabi                 4.0.1                hcfea43d_1    defaults
libedit                   3.1.20181209         hb402a30_0    defaults
libffi                    3.2.1                h475c297_4    defaults
ncurses                   6.1                  h0a44026_1    defaults
openssl                   1.1.1d               h1de35cc_3    defaults
pip                       19.3.1                   py38_0    defaults
python                    3.8.1                h359304d_1    defaults
readline                  7.0                  h1de35cc_5    defaults
setuptools                44.0.0                   py38_0    defaults
sqlite                    3.30.1               ha441bb4_0    defaults
tk                        8.6.8                ha441bb4_0    defaults
wheel                     0.33.6                   py38_0    defaults
xz                        5.2.4                h1de35cc_4    defaults
zlib                      1.2.11               h1de35cc_3    defaults
```


```

Collecting package metadata (current_repodata.json): / done
Solving environment: / failed with initial frozen solve. Retrying with flexible solve.
Solving environment: / failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): \ - - - | \ \ - done
Solving environment: - / | \ - / | \ - / | | \ - / / failed with initial frozen solve. Retrying with flexible solve.
Solving environment: / | | \ - - / | | \ - - / | \ | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
Examining python:  15% 7/48 [00:00<00:00, 3856.58it/s]      
Comparing specs that have this dependency:   0% 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0% 0/7 [00:00<?, ?it/s]
Finding shortest conflict path for python[version='>=2.7,<2.8.0a0']:   0% 0/7 [00:00<?, ?it/s]
Finding shortest conflict path for python=3.8:  14% 1/7 [00:00<00:00, 27.04it/s]              / | - / | \ - / / - - / | \ - - / | \ - / | \ - / / | \ \ - / | \ - - / | | \ \ \ - / | \ - / | \ - - | \ - - / | - - / | \ - / / | \ - / | \ \ / | | \ - / / | \ - | \ \ - / | | - - / / | \ - / | \ \ / | \ - / | \ / / | \ - / / | | \ \ - / \ / \ - | - / \ / | \ - | - | \ / \ / | - / | - | \ / | - | - - | \ - | - | \ - / | - | \ - | \ / \ - | \ - / \ / | \ - 
943
Finding shortest conflict path for python[version='>=3.5,<3.6.0a0']:  29% 2/7 [01:30<03:45, 45.15s/it]
Finding shortest conflict path for python[version='>=3.5,<3.6.0a0']:  43% 3/7 [01:30<02:00, 30.10s/it]
Finding shortest conflict path for python-dateutil:  43% 3/7 [01:30<02:00, 30.10s/it]                 
Finding shortest conflict path for python:  57% 4/7 [01:30<01:30, 30.10s/it]         - / \ - | - | \ / / \ / \ 

Finding shortest conflict path for python[version='>=3.6,<3.7.0a0']:  71% 5/7 [01:36<01:00, 30.10s/it]
Finding shortest conflict path for python[version='>=3.6,<3.7.0a0']:  86% 6/7 [01:36<00:21, 21.73s/it]
Finding shortest conflict path for python[version='>=3.7,<3.8.0a0']:  86% 6/7 [01:36<00:21, 21.73s/it]
Comparing specs that have this dependency:  50% 1/2 [01:37<01:37, 97.03s/it]
Finding conflict paths:   0% 0/2 [00:00<?, ?it/s]
Finding shortest conflict path for python[version='>=3.8,<3.9.0a0']:   0% 0/2 [00:00<?, ?it/s]
Finding shortest conflict path for python=3.8:  50% 1/2 [00:00<00:00, 1669.71it/s]            
Examining setuptools:  19% 9/48 [01:37<07:00, 10.78s/it]
Comparing specs that have this dependency:   0% 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0% 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for setuptools:   0% 0/1 [00:00<?, ?it/s]

Finding conflict paths:   0% 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for setuptools:   0% 0/1 [00:00<?, ?it/s]
Examining certifi:  33% 16/48 [01:37<05:45, 10.78s/it]        
Comparing specs that have this dependency:   0% 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0% 0/4 [00:00<?, ?it/s]
Finding shortest conflict path for certifi:   0% 0/4 [00:00<?, ?it/s]\ - / - / \ \ / \ - | - / - 

Finding shortest conflict path for certifi[version='>=2016.9.26']:  25% 1/4 [00:06<00:19,  6.65s/it]
Finding shortest conflict path for certifi[version='>=2016.9.26']:  50% 2/4 [00:06<00:06,  3.33s/it]- | \ | - | \ - | - - | 

Finding shortest conflict path for certifi[version='>=2016.09']:  50% 2/4 [00:13<00:06,  3.33s/it]  
Finding shortest conflict path for certifi[version='>=2016.09']:  75% 3/4 [00:13<00:04,  4.28s/it]| \ / \ - / \ \ / - | \ | 

Finding shortest conflict path for ca-certificates:  75% 3/4 [00:19<00:04,  4.28s/it]             
Finding shortest conflict path for ca-certificates: 100% 4/4 [00:19<00:00,  5.03s/it]| \ - | / | - / \ / | 

Comparing specs that have this dependency:  50% 1/2 [00:25<00:25, 25.60s/it]

Finding conflict paths:   0% 0/3 [00:00<?, ?it/s]
Finding shortest conflict path for certifi[version='>=2016.9.26']:   0% 0/3 [00:00<?, ?it/s]
Finding shortest conflict path for certifi[version='>=2016.09']:  33% 1/3 [00:00<00:00, 125.26it/s]
Finding shortest conflict path for ca-certificates:  67% 2/3 [00:00<00:00, 126.95it/s]             
Examining wheel:  38% 18/48 [02:02<04:12,  8.40s/it]
Comparing specs that have this dependency:   0% 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0% 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for wheel:   0% 0/1 [00:00<?, ?it/s]| - | - | - / \ | | - / \ / | \ / \ - - | \ - | \ - / \ - | \ / \ - | \ / | - / \ / | \ / | \ / | | - | \ \ / | \ / \ \ / \ - / | - | \ - / \ - \ / | \ - | \ \ - | \ - | - / | \ / \ - | \ / \ - | \ / \ - | \ / \ / | - / | \ - | \ / \ - / \ - / \ / | - | \ - | | - | \ / \ - / / / | - / \ - / / \ - | - / \ / | | - | \ / | - - | \ - / \ - / | \ - | \ - / \ - \ \ | \ - | - / \ - / \ / | \ - / | \ - / \ - | - | \ / | - | \ | \ - / \ - | \ - / - / \ - / \ | 

Comparing specs that have this dependency:  50% 1/2 [02:00<02:00, 120.65s/it]
Finding conflict paths:   0% 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for wheel:   0% 0/1 [00:00<?, ?it/s]
Examining pip:  44% 21/48 [04:03<18:56, 42.08s/it]               
Comparing specs that have this dependency:   0% 0/2 [00:00<?, ?it/s]
Finding conflict paths:   0% 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for pip:   0% 0/1 [00:00<?, ?it/s]\ / \ - \ / \ - | \ / - 

Comparing specs that have this dependency:  50% 1/2 [00:05<00:05,  5.83s/it]
Finding conflict paths:   0% 0/1 [00:00<?, ?it/s]
Finding shortest conflict path for pip:   0% 0/1 [00:00<?, ?it/s]
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Package python-dateutil conflicts for:
matplotlib -> python-dateutil
Package wheel conflicts for:
python=3.8 -> pip -> wheel
matplotlib -> python[version='>=3.5,<3.6.0a0'] -> pip -> wheel
Package pip conflicts for:
matplotlib -> python[version='>=3.5,<3.6.0a0'] -> pip
python=3.8 -> pip
Package ca-certificates conflicts for:
python=3.8 -> openssl[version='>=1.1.1d,<1.1.2a'] -> ca-certificates
matplotlib -> setuptools -> ca-certificates
Package python conflicts for:
python=3.8
Package certifi conflicts for:
python=3.8 -> pip -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
Package setuptools conflicts for:
matplotlib -> setuptools
python=3.8 -> pip -> setuptools
```

**Comment by jjhelmus** [comment]
matplotlib is now available for Python 3.8 for the osx-64 platform.

**Comment by StrikerRUS** [comment]
@jjhelmus Thank you very much for fixing this issue for macOS!
